packages/pwa-kit-dev/src/configs/webpack/overrides-plugin.js

```diff
@@ -42,8 +41,11 @@ class OverridesResolverPlugin {
         const overrideReplace = this.pkg?.ccExtensibility?.overridesDir + '/'
         overridesFsRead.forEach((item) => {
             const end = item.substring(item.lastIndexOf('/index'))
-            const [l, ...rest] = item.split(/(index|\.)/)
-            this.extendsHashMap.set(l?.replace(overrideReplace, '').replace(/\/$/, ''), [end, rest])
+            const [l, ...rest] = item.split(/(index(?!(\.[^.]*\.))|\.(?!([^.]*\.)))/)
+            this.extendsHashMap.set(l?.replace(overrideReplace, '').replace(/\/$/, ''), [
```

Comment:
Is it possible for `l` to be undefined? If it is, then we will have a problem with the map being set to
```
{
  undefined: somevalue
}
```

```suggestion
            this.extendsHashMap.set(l.replace(overrideReplace, '').replace(/\/$/, ''), [
```

Reply:
I took the `?` out @bfeister 

I don't think `l` (I've renamed this to `key`) can be undefined. This piece of code runs inside a forEach where the item we're splitting to get `l` is the import request.